WooCommercePOS/WooCommercePOS.xcodeproj/project.pbxproj

```diff
@@ -614,7 +662,7 @@
 				DEBUG_INFORMATION_FORMAT = "dwarf-with-dsym";
 				ENABLE_NS_ASSERTIONS = NO;
 				ENABLE_STRICT_OBJC_MSGSEND = YES;
-				ENABLE_USER_SCRIPT_SANDBOXING = YES;
+				ENABLE_USER_SCRIPT_SANDBOXING = NO;
```

Comment:
Good question. Since this branches from your protocol abstraction branch, and that one from from Josh's framework, I might have disabled the option after the framework review and since then I haven't merged latest trunk into this branch, so it appears here. 

I believe the diff should disappear once your's is merged into trunk, but I'll keep an eye on it. Currently the framework has both WooCommercePOS and test targets set to `NO`, so it should be fine